dashboard/app/models/levels/bubble_choice.rb

```diff
@@ -143,16 +143,13 @@ def summarize_sublevels(script_level: nil, user_id: nil, should_localize: false)
         level_url(level.id)
 
       if user_id
-        level_info[:perfect] = UserLevel.find_by(
+        user_level = UserLevel.find_by(
           level: level,
           script: script_level.try(:script),
           user_id: user_id
-          )&.perfect?
-        level_info[:status] = if level_info[:perfect]
-                                SharedConstants::LEVEL_STATUS.perfect
-                              else
-                                SharedConstants::LEVEL_STATUS.not_tried
-                              end
+          )
+        level_info[:perfect] = user_level&.perfect?
```

Comment:
Do we still need `level_info[:perfect]`?

Reply:
I saw it's being used here https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/code-studio/components/SublevelCard.jsx#L90 and I didn't bother doing a refactor this time

Reply:
I figured we'd be revisiting lesson extras soon anyway